src/components/Cards/Cards.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Post, Posts } from '../App/App';
+import { Card } from '../Card/Card';
+import './Cards.css';
+
+interface CardsProps {
+  posts: Posts | {};
+}
+
+export const Cards: React.FC<CardsProps> = ({ posts }) => {
+  const makeCards = (posts: any) => {
+    return posts.map((post: Post) => {
+      return <Card key={post.id} post={post} />;
+    });
+  };
+
+  return (
+    <section className='cards-container'>
+      {!!Object.keys(posts).length && makeCards(posts)}
```

Comment:
what is double bang operator for?

Reply:
to remove that pesky 0 if you just do bang and no bang and there are no posts :)